client/src/app.c

```diff
@@ -658,7 +658,7 @@ KeybindHandle app_registerKeybind(int sc, KeybindFn callback, void * opaque, con
     return NULL;
   }
 
-  KeybindHandle handle = (KeybindHandle)malloc(sizeof(struct KeybindHandle));
+  KeybindHandle handle = (KeybindHandle)malloc(sizeof(*handle));
```

Comment:
The original code here made me a little sad, since it leaks the abstraction of what `KeybindHandle` is.

In the original code we had to remember to allocate `sizeof(struct KeybindHandle)` (instead of just `sizeof(KeybindHandle)`), and here we need to allocate `sizeof(*handle)` instead of just `sizeof(handle)`... one has to remember that it's a pointer under the hood.

I propose that separately from this PR, we do away with this typedef (and any others like it).

Reply:
Yes, there is much cleanup to be done across the project like this. Thanks for your efforts so far! I have also been slowly working across the project making such cleanups.